core-blocks/image/test/index.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Internal dependencies
+ */
+import { name, settings } from '../';
+import { blockEditRender } from '../../test/helpers';
+
+describe( 'core/image', () => {
+	test( 'block edit matches snapshot', () => {
+		const wrapper = blockEditRender( name, settings );
+
+		expect( wrapper ).toMatchSnapshot();
```

Comment:
I think this is because `blockEditRender` skips the necessary bootstrapping for `select()` to work, namely `wp.data.registerStore()`. We could export a non-enhanced version of the edit component for testing. Or, we could just not bother 🙂